src/styles/elements/_dropdowns.scss

```diff
@@ -0,0 +1,24 @@
+.dropdown-menu{
+  padding: 8px 10px;
+  border: 1px solid #d3d3d3;
+  font-size: 14px;
```

Comment:
Do you think it's better to use rem or px?

Reply:
Fixed